apps/src/netsim/netsimConstants.js

```diff
@@ -75,3 +76,30 @@ exports.NetSimTabType = {
   ROUTER: 'router',
   DNS: 'dns'
 };
+
+/**
+ * Field types that can go into the packet header.
+ * Map to class-names that can be applied to related input fields.
+ * @enum {string}
+ */
+exports.PacketHeaderType = {
```

Comment:
Hmm... yeah, that's probably the right direction to go, some common `Packet` thing that can be used by the client logs and router logs, etc.  Actually, that's probably the `PacketEncoder`, but constructed with the full packet binary in addition to the format instead of passing the full binary into its methods.  Then it could reasonably own this enum.